lib/remote_files/file.rb

```diff
@@ -79,6 +79,9 @@ def retrieve!
           next unless file
           @content      = file.content
           @content_type = file.content_type
+          if @options[:populate_stored_in]
```

Comment:
This won't raise a KeyError or anything if that value isn't set in the `@options`? If it does then may need to add a check for whether the key exists as well.

Reply:
Also need to add the new option to the list of known keys [here](https://github.com/zendesk/remote_files/pull/26/files#diff-eb648593d3efb23402e60618788bbf355b778db93d29db25762761ec5c743c4fR6)